x-pack/legacy/plugins/maps/public/plugin.ts

```diff
@@ -4,12 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+// @ts-nocheck
 import { Plugin, CoreStart, CoreSetup } from 'src/core/public';
-// @ts-ignore
 import { wrapInI18nContext } from 'ui/i18n';
-// @ts-ignore
 import { MapListing } from './components/map_listing';
-// @ts-ignore
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { setInjectedVarFunc } from '../../../../plugins/maps/public/kibana_services';
```

Comment:
I'd experiment a little here and see if we can have it to work without using `@ts-nocheck`. (e.g. put at end of line)